Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -857,8 +857,19 @@ namespace Babylon
                 }
                 else
                 {
-                    m_position = Napi::Persistent(Napi::Object::New(info.Env()));
-                    m_orientation = Napi::Persistent(Napi::Object::New(info.Env()));
+                    auto position{Napi::Object::New(info.Env())};
+                    position.Set("x", 0.f);
+                    position.Set("y", 0.f);
+                    position.Set("z", 0.f);
+                    position.Set("w", 1.f);
+                    m_position = Napi::Persistent(position);
```

Comment:
Oops, I think I typed this wrong in the suggestion. There should be no `w` in the position.

Reply:
Actually the w is fine / intentional.
See here:
https://github.com/BabylonJS/BabylonNative/blob/0b5676fa5d7b74988fdbee781d81e007759c9b94/Plugins/NativeXr/Source/NativeXr.cpp#L864-L871

A w of 1.0 on a vector like this just means that it's a position and not a normal.

Reply:
Gotcha.